apps/app_manager.py

```diff
@@ -51,99 +54,115 @@ def load_all_apps(self):
                     traceback.print_exc()
                     self.printer_func(["Failed to load", os.path.split(module_path)[1]], self.i, self.o, 2)
         for subdir_path in self.subdir_menus:
-            #Now it's time to link menus to parent menus
+            # Now it's time to link menus to parent menus
             if subdir_path == self.app_directory:
                 continue
             parent_path = os.path.split(subdir_path)[0]
             ordering = self.get_ordering(parent_path)
-            #print("Adding subdir {} to parent {}".format(subdir_path, parent_path))
```

Comment:
So, a proposal about debugging statements like these - add logging, and convert these commented-out prints into debug logs. In the beginning:
```
    import logging
    logger = logging.getLogger(__name__)
    logger.setLevel(logging.INFO)
```
And use `logger.debug()` instead of `print()`